webapp/channels/src/components/interactive_dialog/interactive_dialog.tsx

```diff
@@ -144,15 +154,15 @@ export default class InteractiveDialog extends React.PureComponent {
                 callback_id: callbackId,
                 state,
                 cancelled: true,
-            };
+            }as unknown as DialogSubmission;
```

Comment:
Same here, is there any reason to not provide a proper DialogSubmission object?

Reply:
This is the error i was getting so did like that :p
```
Type 'Record<string, string | number | boolean>' is not assignable to type '{ [x: string]: string; }'.
  'string' index signatures are incompatible.
    Type 'string | number | boolean' is not assignable to type 'string'.
      Type 'number' is not assignable to type 'string'.